app/backend/src/tests/test_api.py

```diff
@@ -63,11 +69,10 @@ def test_ping(db):
     assert res.user.about_me == user.about_me
     assert res.user.my_travels == user.my_travels
     assert res.user.things_i_like == user.things_i_like
+    assert set(language_ability.code for language_ability in res.user.language_abilities) == language_abilities
     assert res.user.about_place == user.about_place
-    # TODO: this list serialisation will be fixed hopefully soon
-    assert res.user.languages == user.languages.split("|")
-    assert res.user.countries_visited == user.countries_visited.split("|")
-    assert res.user.countries_lived == user.countries_lived.split("|")
+    assert res.user.regions_visited == regions_visited
+    assert res.user.regions_lived == regions_lived
```

Comment:
Just hardcode what these are for the test user and remove the calculation of them from the database.